package.json

```diff
@@ -27,17 +27,25 @@
     "watch-ts": "tsc -w",
     "watch-node": "nodemon dist/app.js",
     "watch": "concurrently -k -p \"[{name}]\" -n \"TypeScript, Node\" -c \"yello.bold, cyan.bold\" \"yarn run watch-ts\" \"yarn run watch-node\"",
-    "docs": "apidoc -i src/ -o public/apidoc/"
+    "docs": "apidoc -i src/ -o public/apidoc/",
+    "lint": "yarn eslint . --ext .js,.jsx,.ts,.tsx"
```

Comment:
we need to reduce the scope of this, and as we are just an api on node we don't have jsx files. 
I would suggest this `yarn eslint --ext .js,.ts,.tsx src/** test/**`